qiskit/providers/aer/backends/aer_simulator.py

```diff
@@ -565,6 +567,10 @@ def name(self):
     @classmethod
     def from_backend(cls, backend, **options):
         """Initialize simulator from backend."""
+        if isinstance(backend, BackendV2):
+            raise AerError(
+                "AerSimulator.from_backend is not yet supported for V2 Backends."
```

Comment:
```suggestion
                "AerSimulator.from_backend does not yet support V2 Backends."
```
(I think the grammar is the wrong way round)